connect-id-example/src/main/java/com/telenor/connect/connectidexample/SignInActivity.java

```diff
@@ -3,14 +3,15 @@
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 
 import com.telenor.connect.ConnectCallback;
 import com.telenor.connect.ConnectSdk;
 import com.telenor.connect.ui.ConnectLoginButton;
 import com.telenor.connect.utils.ConnectUtils;
 
+import androidx.appcompat.app.AppCompatActivity;
```

Comment:
Would be nice if you could move the import further up before the com.telenor things